packages/server/src/channels/smooch/conduit.ts

```diff
@@ -14,7 +15,28 @@ export class SmoochConduit extends ConduitInstance<SmoochConfig, SmoochContext>
   public secret!: string
 
   async initialize() {
-    await this.setupWebhook()
+    const target = await this.getRoute()
+    const { webhooks } = await this.smooch.webhooks.list()
+
+    if (yn(process.env.SPINNED)) {
```

Comment:
Why not delete it even if we use an external messaging server?

Reply:
Because that's a useless operation that we don't want to be doing on the messaging server. We only do this so migrations go well. Putting your bot on the cloud is more than a migration.